cluster/environments/azure-multiple-clusters/aks-centralus-variables.tf

```diff
@@ -10,6 +10,10 @@ variable "gitops_central_path" {
   type = "string"
 }
 
+variable "gitops_central_url_branch" {
+  type = "string"
+}
```

Comment:
I would suggest adding a default value to keep configurable values in `.tfvars` at minimum.

Reply:
There is already a default [here](https://github.com/Microsoft/bedrock/blob/master/cluster/common/flux/variables.tf#L24), is that not sufficient to cover all others? Since it's the main flux variable